onBoardProject/screens/login_screen.tsx

```diff
@@ -0,0 +1,121 @@
+import React, {Component, useState} from 'react';
+import {
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+  StyleSheet,
+  Text,
+  TextInput,
+  useColorScheme,
+  View,
+  Pressable,
+  Alert,
+  TouchableOpacity,
+} from 'react-native';
+
+type MyProps = {
+  title: string;
+  email: string;
+  password: string;
+};
+
+function validateEmail(email: any) {
+  const re =
+    /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return re.test(String(email).toLowerCase());
+}
+
+function checkPwd(str: string) {
```

Comment:
acho que aqui ficaria melhor sem abreviação, pode não ficar claro que pwd é password né

```suggestion
function checkPassword(str: string) {
```